### ISSUE TITLE
How to set "batch size"

### ISSUE THREAD

**Issue by lovejing0306** [horovod/horovod#1981]
**Environment:**
1. Framework: TensorFlow
2. Framework version: TensorFlow2.2
3. Horovod version: v0.19.2

**Your question:**
In horovod, the "batch size" is for single gpu or multi gpu. 
thanks.

**Comment by tgaddair** [comment]
Hey @lovejing0306, the batch size you set is the per-worker (single GPU) batch size.  So you should not need to adjust this as you scale up and down the number of workers (instead, we scale the learning rate by `hvd.size()` as shown in the examples).

**Comment by lovejing0306** [comment]
> Hey @lovejing0306, the batch size you set is the per-worker (single GPU) batch size. So you should not need to adjust this as you scale up and down the number of workers (instead, we scale the learning rate by `hvd.size()` as shown in the examples).

thanks
